tofhir-server/src/main/scala/io/tofhir/server/model/InferTask.scala

```diff
@@ -1,11 +1,11 @@
 package io.tofhir.server.model
 
-import io.tofhir.engine.model.{DataSourceSettings, SqlSource}
+import io.tofhir.engine.model.{DataSourceSettings, FhirMappingSourceContext}
 
 /**
  * Infer task instance for getting connection settings and sql query. InferTask object is implemented only for SQL sources.
```

Comment:
Let's fix the comment. In addition to SQL sources, it supports file system as well.